app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
```

Comment:
It's a good practice to provide a docstring for your function to describe what it does, its parameters, its return value, and any exceptions it might raise.